### PR TITLE
chore: manage repo settings

### PR DESCRIPTION
**Pull request by chingor13**
Manage repository settings in code.

### CODE REVIEW

File: .github/sync-repo-settings.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Whether or not rebase-merging is enabled on this repository.
```
Comment (kolea2) [inline .github/sync-repo-settings.yaml:1]: is there any new logic in here? Or just migrating repo settings to this repo?

Comment (chingor13) [reply .github/sync-repo-settings.yaml:1]: It should match the same settings currently on the repository.

Comment (chingor13) [reply .github/sync-repo-settings.yaml:1]: Oh although, maybe we api-bigtable instead of actools as admins (or push?)

**Comment by kolea2** [comment]
Closing as another PR for this was merged

**Pull request closed without merging**